src/__tests__/components/Profile/ProfileEditForm.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { shallow } from '../../../../config/enzymeConfig';
+import { ProfileEditForm } from '../../../components/Profile/ProfileEdit/ProfileEditForm/ProfileEditForm';
+import user from '../../../__mocks__/user';
+import { getMaxListeners } from 'cluster';
+
+
+const props = {
+  editProfile: jest.fn(),
+  user: {},
+}
+describe('<ProfileEditForm />', () => {
+  test('renders without crashing', () => {
+    // const props = { userCredentials: user };
```

Comment:
Please remove this line. You should not have commented code in your PR unless it's a comment. Do this for all other areas with the same issue.